.github/workflows/test.yaml

```diff
@@ -14,6 +14,12 @@ jobs:
       matrix:
         python-version: ["3.7", "3.8", "3.9", "3.10", "3.11"]
         os: [ubuntu-latest, windows-latest]
+        include:
+        - os: ubuntu-22.04
```

Comment:
`ubuntu-latest` is now `ubuntu-24.04`, which does not support Python 3.7. See:

- https://github.com/actions/setup-python/issues/162.